test/functional/fixtures/regression/gh-2775/test.js

```diff
@@ -0,0 +1,155 @@
+const { CLIENT_ERROR_MESSAGE, CLIENT_PAGE_URL, CALLBACK_FUNC_ERROR } = require('./constants');
+const config                                                         = require('../../../config');
+const expect                                                         = require('chai').expect;
+
+
+const expectFailAttempt = (errors, expectedMessage) => {
+    if (Array.isArray(errors))
+        expect(errors[0]).contains(expectedMessage);
+    else {
+        Object.values(errors).forEach(err => {
+            expect(err[0]).contains(expectedMessage);
+        });
+    }
+};
+
+// TODO: fix tests for Debug task
+(config.experimentalDebug ? describe.skip : describe)('[Regression](GH-2775)', () => {
+    describe('TestController command', () => {
+        it('Should skip JS errors with boolean param', async () => {
+            return runTests('./testcafe-fixtures/test-controller.js', 'Should skip JS errors with boolean param');
+        });
+
+        it('Should skip JS errors with only message param', async () => {
+            return runTests('./testcafe-fixtures/test-controller.js', 'Should skip JS errors with only message param');
+        });
+
+        it('Should skip JS errors with multiple options', async () => {
+            return runTests('./testcafe-fixtures/test-controller.js', 'Should skip JS errors with multiple options');
+        });
+
+        it('Should skip JS errors with callback function', async () => {
+            return runTests('./testcafe-fixtures/test-controller.js', 'Should skip JS errors with callback function');
+        });
+
+        it('Should fail due to incorrect message option', async () => {
```

Comment:
Bad naming here and later.
Should be something like : `Should fail if the 'message' option doesn't satisfy the client error message'`.